src/library/scala/math/package.scala

```diff
@@ -215,9 +218,37 @@ package object math {
     */
   def signum(x: Long): Long     = java.lang.Long.signum(x)
   /** @group signum */
-  def signum(x: Float): Float   = java.lang.Math.signum(x)
+  def signum(f: Float): Float   = java.lang.Math.signum(f)
   /** @group signum */
-  def signum(x: Double): Double = java.lang.Math.signum(x)
+  def signum(d: Double): Double = java.lang.Math.signum(d)
+
+  def floorDiv(x: Int, y: Int): Int = java.lang.Math.floorDiv(x, y)
+
+  def floorDiv(x: Long, y: Long): Long = java.lang.Math.floorDiv(x, y)
+
+  def floorMod(x: Int, y: Int): Int = java.lang.Math.floorMod(x, y)
+
+  def floorMod(x: Long, y: Long): Long = java.lang.Math.floorMod(x, y)
+
+  def copySign(magnitude: Double, sign: Double): Double = java.lang.Math.copySign(magnitude, sign)
```

Comment:
@golem131 `copySign` can be included in the "Signs" group. The group name is `signum` so rename that to `@group signs` to make it less specific to `signum`.